app/models/order.rb

```diff
@@ -22,4 +22,8 @@ def change_status_packaged
       update_attribute(:order_status, 'packaged')
     end
   end
+
+  def merchant_items(merch_id)
+    items.select("items.id, items.name, items.image, items.price, items.inventory, item_orders.order_status, item_orders.quantity, item_orders.id as io_id").joins(:item_orders).where(merchant_id: merch_id)
+  end
```

Comment:
Joins the two tables to get only the information needed as opposed to all columns from both tables. Renamed item_orders.id to io_id to not confused with item.id